pkgs/applications/networking/seahub/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib, fetchFromGitHub, python3, makeWrapper }:
+let
+  # Seahub 8.x.x does not support django-webpack-loader >=1.x.x
+  python = python3.override {
+    packageOverrides = self: super: {
+      django-webpack-loader = super.django-webpack-loader.overridePythonAttrs (old: rec {
+        version = "0.7.0";
+        src = old.src.override {
+          inherit version;
+          sha256 = "0izl6bibhz3v538ad5hl13lfr6kvprf62rcl77wq2i5538h8hg3s";
+        };
+      });
+    };
+  };
+in
+python.pkgs.buildPythonApplication rec {
+  pname = "seahub";
+  version = "8.0.8";
+
+  src = fetchFromGitHub {
+    owner = "haiwen";
+    repo = "seahub";
+    rev = "c51346155b2f31e038c3a2a12e69dcc6665502e2"; # using a fixed revision because upstream may re-tag releases :/
```

Comment:
Was upstream made aware of this problem? I remember that they retagged a version of seafile-client back when I maintained its Nix package and their excuse was that it was some intern's mistake or so.

Reply:
I'm afraid upstream does not really care. We could try to ask about this problem in particular again, but in general the last time I asked for updating the outdated build instructions there was no response at all.
Also, they've completely shifted to docker-only deployments, so unless for development use they likely see no interest in building stuff from source.

Reply:
This is unfortunately [a reported issue](https://github.com/haiwen/seafile/issues/2323)... Upstream also tend to tag [before the actual release](https://forum.seafile.com/t/change-experimental-repo-tags/6202).